### ISSUE TITLE
Last replied time for message

### ISSUE THREAD

**Issue by temiotu** [uvdesk/api-bundle#18]
Hi 
Is it possible to retrieve just headline information about a ticket without pulling all the threads etc?

For instance, we would like to retrieve the last time a new response was posted on a ticket by anyone.

**Comment by papnoisanjeev** [comment]
@temiotu 

Do you want a last reply time on ticket listing for every ticket ?

**Comment by temiotu** [comment]
@papnoisanjeev  yes but i was hoping maybe i could get that in the metadata endpoint rather than when i get the ticket. I just want to be get information about the ticket without retrieving it and its threads because thats alot of information i don't need.
